services/resource-usage-tracker/.env-devel

```diff
@@ -0,0 +1,8 @@
+RESOURCE_USAGE_TRACKER_DEV_FEATURES_ENABLED=1
```

Comment:
This is off on master, staging, prod? Please add a quick note about this env-var and desired settings in the PR notes, thx

Reply:
I would turn it on, on all deployments. I will add a note :+1: